test/mocks/http/mocks.h

```diff
@@ -201,6 +201,18 @@ class MockStreamDecoderFilterCallbacks : public StreamDecoderFilterCallbacks,
   bool stream_destroyed_{};
 };
 
+// A version of MockStreamDecoderFilterCallbacks that mocks sendLocalReply()
+// TODO(mattklein123): Get rid of this and fix any tests to work with this function mocked in the
+//                     main class.
+class MockStreamDecoderFilterCallbacksWithMockedSendLocalReply
+    : public MockStreamDecoderFilterCallbacks {
+public:
+  MOCK_METHOD5(sendLocalReply, void(Code code, absl::string_view body,
```

Comment:
Can you just mock this in `MockStreamDecoderFilterCallbacks` and move its current default implementation to `ON_CALL(...).ByDefault(Invoke)...`? Other mocks does similar things.

Reply:
Yeah this is what I was going to fix in the follow up since I wasn't sure what would break, but I will just fix it now.